Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -1,72 +1,111 @@
+}
+
+function handleRollupError(error) {
+    const stack = error?.stack?.split(/[\n\r]+/).map((line) => line.trim()) ?? [];
+    const extensionRegex = /\.(js|vue):(\d+)(:(\d+))?.*/;
+    const pathLine = stack
+        .filter((line) => line.match(extensionRegex))
+        .map((line) => line.replace(/^\s*at\s+/, '').trim())[0];
+    const lineMatch = pathLine?.match(extensionRegex) ?? [];
+
+    handleErrorObject({
+        code: 'ROLLUP',
+        path: pathLine?.replace(extensionRegex, '.js'),
+        line: lineMatch ? lineMatch[2] : undefined,
+        column: lineMatch ? lineMatch[4] : undefined,
+        message: error,
+    });
+}
+
+// Used for setting defaults and preventing it would be needlessly complicated.
+/* eslint-disable no-param-reassign */
 
 module.exports = function rollupPipeline(
     destinationPath,
     filesAndEntryPaths,
     rollupPlugins,
     rollupOptions = null,
     outputFileNameTransform = null) {
-    return all(filesAndEntryPaths
-        .map((pair) => {
-            const fileName = pair.fileName;
-            const entryPath = pair.entryPath;
-            const outputFileName = outputFileNameTransform ? outputFileNameTransform(fileName) : fileName;
-
-            const defaultRollupOptions = {
-                allowRealFiles: true,
-                input: path.join(entryPath),
-                output: {
-                    format: 'cjs',
-                },
-                onwarn: (warning, next) => {
-                    if (warning.code === 'THIS_IS_UNDEFINED') return;
-                    next(warning);
-                },
-            };
-
-            if (typeof rollupOptions === 'function') rollupOptions = rollupOptions(fileName, entryPath);
-            if (typeof rollupPlugins === 'function') rollupPlugins = rollupPlugins(fileName, entryPath);
-
-            const options = rollupOptions ? { ...defaultRollupOptions, ...rollupOptions } : defaultRollupOptions;
-
-            if (Array.isArray(rollupPlugins)) {
-                if (Array.isArray(options.plugins)) {
-                    rollupPlugins.forEach(plugin => options.plugins.push(plugin));
-                }
-                else {
-                    options.plugins = rollupPlugins;
-                }
+    function configure(fileName, entryPath) {
+        const defaultRollupOptions = {
+            onwarn: (warning, next) => {
+                if (warning.code === 'THIS_IS_UNDEFINED') return;
+                next(warning);
+            },
+        };
+
+        if (typeof rollupOptions === 'function') rollupOptions = rollupOptions(fileName, entryPath);
+        if (typeof rollupPlugins === 'function') rollupPlugins = rollupPlugins(fileName, entryPath);
+
+        const options = rollupOptions ? { ...defaultRollupOptions, ...rollupOptions } : defaultRollupOptions;
+        options.input = entryPath;
+
+        if (Array.isArray(rollupPlugins)) {
```

Comment:
Please add a comment for my future self:
```suggestion
        // Safely copy the provided plugins to the options.
        if (Array.isArray(rollupPlugins)) {
```